src/Tizen.Multimedia.Vision/MediaVision/InferenceFaceDetector.cs

```diff
@@ -0,0 +1,208 @@
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public InferenceFaceDetectorResult Inference(MediaVisionSource source)
+        {
+            ValidateNotDisposed();
+
+            if (source == null)
+            {
+                throw new ArgumentNullException(nameof(source));
+            }
+
+            InteropFD.Inference(_handle, source.Handle).Validate("Failed to inference face detection.");
+
+            return new InferenceFaceDetectorResult(_handle);
+        }
+
+        /// <summary>
+        /// Detects faces on the source image asynchronously.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="InferenceFaceDetectorResult.BoundingBoxes"/> can be empty, if there's no detected face.<br/>
+        /// This method uses about twice as much memory as <see cref="InferenceFaceDetector.Inference"/>.
+        /// </remarks>
+        /// <param name="source">The image data to detect faces.</param>
+        /// <exception cref="ObjectDisposedException">The InferenceFaceDetector already has been disposed.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public async Task<InferenceFaceDetectorResult> InferenceAsync(MediaVisionSource source)
+        {
+            ValidateNotDisposed();
+
+            if (source == null)
+            {
+                throw new ArgumentNullException(nameof(source));
+            }
+
+            InteropFD.InferenceAsync(_handle, source.Handle).Validate("Failed to inference face detection.");
+
+            return await Task.Factory.StartNew(() => new InferenceFaceDetectorResult(_handle),
+                CancellationToken.None,
+                TaskCreationOptions.DenyChildAttach | TaskCreationOptions.LongRunning,
+                TaskScheduler.Default);
+        }
+
+        private ulong _requestId = 1;
+        /// <summary>
+        /// Requests detecting faces to get their bounding boxes asynchronously.
+        /// </summary>
+        /// <remarks>
+        /// This function does not guarantee that inference is done when this method returns. The user can get the result by using <see cref="GetRequestResults"/>.<br/>
+        /// If the user calls this method again before the previous one is finished internally, the call will be ignored.<br/>
+        /// <see cref="InferenceFaceDetectorResult.BoundingBoxes"/> can be empty, if there's no detected face.<br/>
+        /// Note that this method could use about twice as much memory as <see cref="InferenceFaceDetector.Inference"/>.
+        /// </remarks>
+        /// <param name="source">The image data to detect faces.</param>
+        /// <returns>The request id that indicates the order of request.</returns>
```

Comment:
please replace all "request id" with "request ID"
IMO it would sound better as "order of requests".

Reply:
I'll fix it.